dept44-example/src/main/java/se/sundsvall/petinventory/api/PetInventoryResource.java

```diff
@@ -41,34 +41,36 @@ public class PetInventoryResource {
 
 	private final PetInventoryService petInventoryService;
 
-	public PetInventoryResource(PetInventoryService petInventoryService) {
+	public PetInventoryResource(final PetInventoryService petInventoryService) {
```

Comment:
While you're at it: 
Remove `public` visibility modifier from class and constructor?